### PR TITLE
Remove raiseContainerWarning from IContainerContext in next branch

### PR DESCRIPTION
**Pull request by sonalivdeshpande**

### CODE REVIEW

**Comment by sdeshpande3** [comment]
not sure why there are build failures with container-definitions, builds fine locally

File: common/lib/container-definitions/package.json [hunk 1]
```diff
@@ -519,11 +519,6 @@
           "backCompat": false,
           "forwardCompat": false
         }
-      },
-      "0.47.1000": {
```
Comment (heliocliu) [inline common/lib/container-definitions/package.json:523]: this is probably what's causing the break

Comment (sdeshpande3) [reply common/lib/container-definitions/package.json:523]: It breaks locally when this is added back in

**Review by heliocliu — APPROVED** [review]

File: BREAKING.md [hunk 2]
```diff
@@ -20,21 +20,27 @@ There are a few steps you can take to write a good change note and avoid needing
...
 # 0.58
 
 ## 0.58 Upcoming changes
 - [Doing operations not allowed on deleted sub directory](#Doing-operations-not-allowed-on-deleted-sub-directory)
+- [raiseContainerWarning removed from IContainerContext](#raiseContainerWarning-removed-from-IContainerContext)
 
 ### Doing operations not allowed on deleted sub directory
 Users will not be allowed to do operations on a deleted directory. Users can subscribe to `disposed` event to know if a sub directory is deleted. Accessing deleted sub directory will throw `UsageError` exception now.
 
+### raiseContainerWarning removed from IContainerContext
```
Comment (heliocliu) [inline BREAKING.md:41]: this part of the change should go into `main` since that's where partner teams are looking for this info

**Pull request merged**